measures/UpgradeCosts/tests/MF_1story_UB_Furnace_AC1_FuelTankWH.osw

```diff
@@ -126,6 +126,7 @@
         "geometry_unit_front_wall_is_adiabatic": true,
         "geometry_unit_back_wall_is_adiabatic": false,
         "geometry_building_num_units": 8,
+        "geothermal_loop_configuration": "none",
```

Comment:
Err, why was this made a required input? Change it to be optional like all the other geothermal loop arguments. If not provided, it should clearly behave the same as "none".

Reply:
Sure, we can do it that way, too. https://github.com/NREL/OpenStudio-HPXML/pull/1574